lang/py/src/avro/schema.py

```diff
@@ -33,6 +33,10 @@
   A boolean; or
   Null.
 """
+from abc import ABCMeta, abstractmethod
```

Comment:
Since we don't have interfaces in python, to make it OO I read that this is a good way to do it. Imho I think this makes it obvious that you cannot instantiate the base class and when you inherit it, it also makes the protocol clear. Unless am missing something, I vote we keep it this way.